test/e2e/storage/testsuites/disruptive.go

```diff
@@ -38,7 +38,6 @@ func InitDisruptiveTestSuite() TestSuite {
 			name:       "disruptive",
 			featureTag: "[Disruptive]",
 			testPatterns: []testpatterns.TestPattern{
-				testpatterns.DefaultFsInlineVolume,
```

Comment:
I believe that this pattern should not be deleted.

Reply:
whoops fixing